.github/workflows/cd-to-infra.yml

```diff
@@ -46,6 +46,12 @@ jobs:
           # Use the release tag to deploy, if one is available.
           DEPLOY_TAG=$RELEASE_TAG
         fi
+        # Tag with latest if this is a release (not pre-release)
```

Comment:
Can you switch this check around? Should tag `latest` only if this is a push to `main`, versus for a release.

Reply:
updated!